src/types/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum ErrorMessages {
+  unableToloadTodos = 'Unable to load todos',
+  emptyTitle = 'Title should not be empty',
+  unableToAddTodo = 'Unable to add a todo',
+  unableToDelete = 'Unable to delete a todo',
+  unableToUpdate = 'Unable to update a todo',
```

Comment:
please write with the first Upper case letter

`emptyTitle`  change to `EmptyTitle`

![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/91946036/ed8dacf9-54d8-4589-9ba8-adbaf2b56baa)